### DISCUSSION TITLE
Verified Student but don’t have access to benefits

### DISCUSSION THREAD

**Discussion by jvbraganca** [community/community#66772] [GitHub Education] [answered]
### Hi everyone,

Hi,

I re-verified my student developer pack on September 8th, and it was accepted. But, since then I do not have access to any benefits. How long it takes before the benefits are available? Before, it was automatic.

**Comment by arafajs** [comment]
According to the GitHub documentation, the activation process of benefits for the GitHub Student Developer Pack usually takes up to 72 hours. However, it may take longer during peak times, such as during the start of a new semester.

**Reply by raniagomaa123456** [reply]
I have already been approved for the get Hub student package, so whenever I try to book a Mongo certificate, this message appears to me

Welcome and thanks for logging in. Unfortunately we are unable to verify your status as a participant in the GitHub Student Developer Pack. Please visit the GitHub Student Pack home for additional information and how to register.

Approved Submitted Aug 31, 2023
Congrats! Your request for [@raniagomaa123456](https://github.com/raniagomaa123456) was approved and your benefits are now available.

**Reply by samus-aran** [reply]
Hi @raniagomaa123456 👋 

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍

**Comment by mantasu** [comment, marked as answer]
Hey, I had the same issue, it is officially stated that verification can take a few weeks; I had to wait as well until "Approved" tag became purple:

![Screenshot 2023-09-24 072637](https://github.com/community/community/assets/56790921/d80b71bb-8d34-424d-a682-39a0f986389c)

But if it's been 2-3 weeks and you still havent's received the benefits, I would suggest just opening a [ticket](https://support.github.com/request/education?tags=docs-generic).

**Reply by raniagomaa123456** [reply]
I have already been approved for the get Hub student package, so whenever I try to book a Mongo certificate, this message appears to me

Welcome and thanks for logging in. Unfortunately we are unable to verify your status as a participant in the GitHub Student Developer Pack. Please visit the GitHub Student Pack home for additional information and how to register.

Approved Submitted Aug 31, 2023
Congrats! Your request for [@raniagomaa123456](https://github.com/raniagomaa123456) was approved and your benefits are now available.

**Comment by advayc** [comment]
You'll get your benefits about 72 hours after your application. After then you'll get a email along with the PRO badge on your github profile.

**Reply by raniagomaa123456** [reply]
I have already been approved for the get Hub student package, so whenever I try to book a Mongo certificate, this message appears to me

Welcome and thanks for logging in. Unfortunately we are unable to verify your status as a participant in the GitHub Student Developer Pack. Please visit the GitHub Student Pack home for additional information and how to register.

Approved Submitted Aug 31, 2023
Congrats! Your request for [@raniagomaa123456](https://github.com/raniagomaa123456) was approved and your benefits are now available.

**Reply by raniagomaa123456** [reply]
@advay-c

**Reply by samus-aran** [reply]
Let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍 Also don't **mention/tag** other community members randomly.

**Comment by LemonLamo** [comment]
I am a verified student but still dont have access to benefits such a github Copilot , its still telling me to pay , what can i do ?

**Reply by samus-aran** [reply]
Hi @jvbraganca  👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍

**Comment by JoyciPM** [comment]
I was approved, but I keep receiving this message:

"Welcome and thanks for logging in. Unfortunately, we are unable to verify your status as a participant in the GitHub Student Developer Pack. Please visit the GitHub Student Pack home for additional information and how to register."

I've already completed courses on MongoDB and received a 50% discount.

Could you please help me understand why this message still appears even after my approval?

**Reply by samus-aran** [reply]
Hi @JoyciPM  👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍
